flowdb/bin/build/002_tweak_config.sh

```diff
@@ -24,5 +24,14 @@
 #
 set -e
 
+# Allows WAL to be skipped for CREATE TABLE & CLUSTER
+# we turn this on in the autogenerated config anyway, but nice
+# to have it off from the start for the benefit of the testdata containers.
```

Comment:
```suggestion
# to have it off from the start for the benefit of the testdata containers
# for increased performance during startup.
```
(That's the benefit as I understand it? Might be nice adding a half-sentence to indicate why that's the case.)